### PR TITLE
SearchKit - Use POST to send contact ids to action forms

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Fixes [dev/core#2736](https://lab.civicrm.org/dev/core/-/issues/2736), allows large numbers of rows to be bulk-updated by legacy search actions in SearchKit.

### CODE REVIEW

File: ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php [hunk 1]
```diff
@@ -41,7 +41,8 @@ public function _run(\Civi\Api4\Generic\Result $result) {
         'icon' => 'fa-file-excel-o',
         'crmPopup' => [
           'path' => "'civicrm/export/standalone'",
-          'query' => "{reset: 1, entity: '{$entity['name']}', id: ids.join(',')}",
+          'query' => "{reset: 1, entity: '{$entity['name']}'}",
+          'data' => "{id: ids.join(',')}",
```
Comment (colemanw) [inline ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php:45]: @eileenmcnaughton this PR addresses all actions but export - we need to add qfKey here but need a function that returns the quickform export class name given an entity name. Can't find one...

Comment (colemanw) [reply ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php]: Nevermind I figured it out.

**Comment by eileenmcnaughton** [comment]
@demeritcowboy are you able to check this out? would value your eyes on since my last merge on this got reverted :-)

**Comment by colemanw** [comment]
@demeritcowboy no, this adds a new param to `crmSnippet` - it will only use post if you set that param.

**Comment by eileenmcnaughton** [comment]
woohoo

**Comment by eileenmcnaughton** [comment]
See https://lab.civicrm.org/dev/core/-/issues/3826 for related regression

**Pull request merged**